tensorflow/go/op/scope_test.go

```diff
@@ -95,6 +95,40 @@ func TestMultipleGeneratedOps(t *testing.T) {
 	}
 }
 
+func TestScopeWithGraph(t *testing.T) {
+	s1 := NewScope()
+
+	var hello interface{} = "hello"
```

Comment:
Why define `hello` as an `interface{}` instead of just `string`?

This would work just as well, no?
```go
hello := "hello"
```